src/resources/elements/primeDesignSystem/pform-input/pform-input.scss

```diff
@@ -1,17 +1,20 @@
 @import "../styles";
 
+$pformInputLineHeight: 33px;
+
 pform-input {
   display: block;
+
   .formInputLabel {
     display: flex;
     color: $Secondary02;
-    margin-bottom: $spacing-normal;
     align-items: center;
     justify-content: space-between;
 
     .formInputLabelTitle {
       font-size: $font-size3;
       font-weight: 700;
+      line-height: $pformInputLineHeight;
```

Comment:
Had to do some general adjustment to the styles, because they were out of sync with Mocks.
And changes were related to "new line" change